<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSettingDialog</class>
 <widget class="QDialog" name="CameraSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraSettingDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
border-image: url(:/SeedUI/Resources/SeedUI/PC_Camera.png);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget
{
	
	background-color: rgb(34, 36, 40);
}

QCheckBox
{
	color: rgb(85, 85, 85);
	font: 75 14px &quot;微软雅黑&quot;;
	
}

QCheckBox::indicator:unchecked 
{
	image: url(:/SeedUI/Resources/SeedUI/NoCancel.png);
}

QCheckBox::indicator:checked 
{
	image: url(:/SeedUI/Resources/SeedUI/Cancel.png);
}


QCheckBox::indicator:unchecked:hover
{
	image: url(:/SeedUI/Resources/SeedUI/NoCancel_H.png);
}

QCheckBox::indicator:checked:hover
{
	image: url(:/SeedUI/Resources/SeedUI/Cancel_H.png);
}</string>
     </property>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>2</y>
        <width>452</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget_2
{
	
	background-color: rgb(38, 40, 45);
}</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>9</y>
         <width>15</width>
         <height>14</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/SeedUI/Resources/SeedUI/SeedUI_Camera.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>8</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(234, 126, 24);</string>
       </property>
       <property name="text">
        <string>摄像头设置</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Close_pushButton">
       <property name="geometry">
        <rect>
         <x>435</x>
         <y>8</y>
         <width>10</width>
         <height>10</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Close_pushButton{
	
	border-image: url(:/Close/Resources/Close_Normal.png);
}
QPushButton#Close_pushButton:hover{
	
	border-image: url(:/Close/Resources/Close_Hover.png);
}
QPushButton#Close_pushButton:disabled{
	
	border-image: url(:/Close/Resources/Close_Disabled.png);
}
QPushButton#Close_pushButton:checked{
	
	border-image: url(:/Close/Resources/Close_checked.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>226</y>
        <width>452</width>
        <height>93</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget_3
{
	
	background-color: rgb(38, 40, 45);
}</string>
      </property>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>60</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#OK_pushButton
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButtonYellow.png);
	font: 12px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
}

QPushButton#OK_pushButton:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButtonYellow_H.png);
}

QPushButton#OK_pushButton:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButtonYellow_C.png);
}
</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>60</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#Cancel_pushButton
{
	font:  12px &quot;微软雅黑&quot;;
	color: rgb(240, 240, 240);
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButton.png);
}

QPushButton#Cancel_pushButton:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButton_H.png);
}

QPushButton#Cancel_pushButton:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/SmallButton_C.png);
}
</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 85, 85);
font: 75 14px &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>摄像头选择</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>261</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>261</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">/**********下拉列表**********/
QComboBox {
    background-color: black;
    border: 1px solid #333333;
    color: white;
    padding:1px 2em 1px 3px;
	font: 75 14px &quot;微软雅黑&quot;;
	border-radius:5px;
}
QComboBox:hover {
    background-color: rgb(24, 24, 24);
    border: 1px solid #333333;
    color: white;
    padding:1px 2em 1px 3px;
	font: 75 14px &quot;微软雅黑&quot;;
	border-radius:5px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    border-left: 1px solid #333333;
	width: 25px;
}/* 下拉按钮 */
QComboBox::drop-down:hover {
  	subcontrol-origin: padding;
    subcontrol-position: top right;
    border-left: 1px solid #333333;
	background-color: rgb(83, 83, 83);
}/* 下拉按钮 鼠标移动 */
QComboBox::down-arrow {
    border: 1px solid #333333;
    width: 8px;
    height: 5px;
    background: rgb(83, 83, 83);
	border-bottom-left-radius: 5px; 
	border-bottom-right-radius: 5px; 
}/* 下拉箭头 */
QComboBox::down-arrow:hover {
    border: 1px solid #333333;
    width: 8px;
    height: 5px;
    background: rgb(170, 170, 170);
	border-bottom-left-radius: 5px; 
	border-bottom-right-radius: 5px; 
}/* 下拉箭头 鼠标移动 */
QComboBox QAbstractItemView{
	outline:0px;/*去掉虚线*/
	color: rgb(209, 209, 209);
	font: 75 14px &quot;微软雅黑&quot;;
	background-color: rgb(43, 43, 43);
	 selection-background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffa02f, stop: 1 #d7801a);
}/* 下拉选择 */
QComboBox QAbstractItemView::item{height:20px;}/*下拉选择高度*/</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Refresh_pushButton">
      <property name="geometry">
       <rect>
        <x>355</x>
        <y>58</y>
        <width>83</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#Refresh_pushButton
{
	font:  12px &quot;微软雅黑&quot;;
	color: rgb(220, 220, 220);
	border-image: url(:/SeedUI/Resources/SeedUI/DownloadButton.png);
}

QPushButton#Refresh_pushButton:hover
{
	border-image: url(:/SeedUI/Resources/SeedUI/DownloadButton_H.png);
}

QPushButton#Refresh_pushButton:checked
{
	border-image: url(:/SeedUI/Resources/SeedUI/DownloadButton_C.png);
}
</string>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>411</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 85, 85);
font: 75 14px &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>画面美白</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QSlider" name="horizontalSlider">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider  
{  
    background-color: rgb(34, 36, 40);
    border-style: outset;   
    border-radius: 10px;   
}  
  
QSlider::groove:horizontal  
{  
    height: 2px;  
    background: rgb(10,10,10);  
    margin: 2px 0  
}  

QSlider::handle:horizontal   
{  
    background: rgb(254,161,24);  
    width: 10px;  
    height:2px;  
    margin: -5px 0px -5px px;  
    border-radius:5px;   
    
    border:1px solid rgb(254,161,24);  
} </string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="Whitening_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 85, 85);
font: 75 14px &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>+50</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="checkBox_3">
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 85, 85);
font: 75 14px &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>开</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 85, 85);
font: 75 14px &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>镜头方向</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="checkBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>水平翻转</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(85, 85, 85);
font: 75 14px &quot;微软雅黑&quot;;</string>
         </property>
         <property name="text">
          <string>垂直翻转</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rdzbdemo.qrc"/>
 </resources>
 <connections/>
</ui>
